source/vmix_kpp.F90

```diff
@@ -100,10 +99,10 @@ module vmix_kpp
       lccsm_control_compatible !flag for backwards compatibility with ccsm4 control
 
    character (char_len) :: &
-      langmuir_opt          ! QL, 150526, lanmguir mixing parameterization option
-                            ! valid: null, vr12-ma, vr12-en
+      langmuir_opt          ! lanmguir turbulence parameterization option
+                            ! valid: null, vr12-ma, lf17
```

Comment:
It looks like these valid values aren't captured in `namelist_definition_pop.xml`. I think you should update that file:

```
<entry
id="langmuir_opt"
type="char*256"
category="Vertical Mixing (KPP)"
- group="vmix_kpp_nml" >
- Langmuir mixing parameterization option.
+ group="vmix_kpp_nml"
+ valid_values="null,vr12-ma,lf17" >
+ Langmuir turbulence parameterization option.

- Valid Values: 'null', 'vr12-ma', 'vr12-en'
+ Valid Values: 'null', 'vr12-ma', 'lf17'
Default: 'vr12-ma'
</entry>
```

(Maybe you want to continue to support `vr12-en` as well? If so, modifiy appropriately.)

Reply:
`vr12-en` is no longer supported in the new CVMix tag. I don't think anyone has actually used it as it is an early version of the Langmuir entrainment parameterization.